firestore/src/android/firestore_android.h

```diff
@@ -216,8 +216,8 @@ class FirestoreInternal {
 #endif  //  defined(_STLPORT_VERSION)
 
   Mutex bundle_listeners_mutex_;
-  std::vector<std::unique_ptr<LambdaEventListener<LoadBundleTaskProgress>>>
-      bundle_listeners_;
+  // Using a vector to ensure FirestoreInternal owns all listener instance.
```

Comment:
Optional: consider `to ensure listener instances cannot outlive FirestoreInternal`.

Reply:
Done.